src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 3 failures in `PowerShell-CI-windows`
Test-Connection.Ping.Default parameter set is 'Ping'
```powershell
Expected the value to have type [System.Net.NetworkInformation.PingOptions] or any of its subtypes, but got $null with type $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Test-Connection.Tests.ps1: line 41
41:             $result[0].Options | Should -BeOfType "System.Net.NetworkInformation.PingOptions"
```
Test-Connection.Ping.BufferSize Should -Be between 0 and 65500
```powershell
Parameter set cannot be resolved using the specified named parameters. One or more parameters issued cannot be used together or an insufficient number of parameters were provided.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Test-Connection.Tests.ps1: line 152
152:             { Test-Connection $targetName -BufferSize 0 } | Should -Not Throw

```
Test-Connection.Ping.ResolveDestination for name
```powershell
Expected exactly 'fe80::5dab:af49:74cf:7591', but got ::1.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Test-Connection.Tests.ps1: line 184
184:             $result.Address | Should -BeExactly $resolvedAddress
```